backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -14,11 +15,29 @@ public class PermissionService(
 {
     private LexAuthUser? User => loggedInContext.MaybeUser;
 
+    private async ValueTask<bool> ManagesOrgThatOwnsProject(Guid projectId)
+    {
+        var project = await dbContext.Projects.Include(p => p.Organizations).Where(p => p.Id == projectId).FirstOrDefaultAsync();
```

Comment:
I'd like to pull this and any other query out into the `ProjectService` and then cache the result as I'd like to avoid executing this query every time a request is made during S&R. Be sure to only query the limited data that's needed, and not the full project and org objects.